packages/fluentui/react-builder/src/components/NarrationComputer.tsx

```diff
@@ -0,0 +1,268 @@
+/*
+ TODO:
+* Add the missing and not so obvious attributes (e.g. "aria-haspopup" or "aria-expanded") for the already defined roles (e.g. "menuitem" or "checkbox") according to the specification.
+* With JAWS, in the case of the element with the "listbox" role, differentiate between having and not having the aria-multiselectable="true" attribute. If this attribute is present, then aria-selected="false" on the child elements with the role "option" behave differently than if it is not present. Specifically, if aria-multiselectable="true" is present, the aria-selected="false" causes the narration of "not selected", but if present, having the aria-selected attribute makes no difference to the narration. 
+ * Should we also consider the "disabled" state?
+ */
+import SRNC from './SRNC-Definitions';
+import './SRNC-Rules-Win_JAWS';
+
+export default class NarrationComputer {
+  private computedParts: Record<string, string> = {
```

Comment:
```suggestion
  computedParts: Record<string, string> = {
```
[avoid visibility modifiers (public, protected, private)](https://github.com/microsoft/fluentui/blob/master/packages/eslint-plugin/README.md#no-visibility-modifiers)

Reply:
Okay, can I ask why they should be removed?

Reply:
https://github.com/microsoft/fluentui/blob/master/packages/eslint-plugin/README.md#no-visibility-modifiers